src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Cool, but you can unit this logic into one block

Reply:
Thanks, ok  i understood